inventory/sample.aws.example.com.d/inventory/group_vars/all.yml

```diff
@@ -11,31 +11,76 @@ hosting_infrastructure: aws
 # Cluster Environment ID to uniquely identify the environment
 env_id: "<REPLACE WITH VALID ENV ID - i.e: env1>"
 
-## AWS Provisioning Variables
-# Amazon image name to use for instances.
-# Note that each AWS region may utilize a different AMI for the same image.
-aws_image_name: <REPLACE WITH VALID AMI>
-
-# AWS instance flavors (i.e.: instance sizes)
-master_flavor: m4.xlarge
-infra_node_flavor: i3.xlarge
-app_node_flavor: m4.xlarge
-cns_node_flavor: i3.xlarge
-
-# Number of master instances to deploy.
-# This is a non-HA sample nventory so don't change this value
-aws_num_masters: 1
-
-# Number of instances to be used as infra nodes.
-# This is a non-HA sample inventory so don't change this value
-aws_num_infra_nodes: 1
-
-# Number of instances to be used as compute/app nodes
-aws_num_app_nodes: 3
-
-# Number of instances to be used as CNS (Container Native Storage) nodes
-aws_num_cns_nodes: 0
-
+# Define custom Master(s) API adn Console Port - Default 8443
+#openshift_master_api_port: 443
+#openshift_master_console_port: 443
+
+# Define custom console and apps public DNS Name
+# NOTE: These values need to be a subset of {{ dns_domain }}
+openshift_master_default_subdomain: "apps.{{ env_id }}.{{ dns_domain }}"
+openshift_master_cluster_hostname: "console.internal.{{ env_id }}.{{ dns_domain }}"
+openshift_master_cluster_public_hostname: "console.{{ env_id }}.{{ dns_domain }}"
+
+# Define infrastructure skeleton
+cloud_infrastructure:
+   region: <REPLACE WITH VALID REGION>
+   image_name: <REPLACE WITH VALID AMI>
+   masters:
+     count: <REPLACE WITH NUMBER OF INSTANCES TO CREATE>
+     flavor: <REPLACE WITH DESIRED FLAVOR FOR THE INSTANCE>
+     zones:
+     - <REPLACE WITH THE AZ(s) LIST TO PLACE THE INSTANCES IN>
+     - <REPLACE WITH THE AZ(s) LIST TO PLACE THE INSTANCES IN>
+     - <REPLACE WITH THE AZ(s) LIST TO PLACE THE INSTANCES IN>
+     name_prefix: <REPLACE WITH DESIRED NAME PREFIX FOR THE INSTANCE>
+     root_volume_size: <REPLACE WITH THE SIZE (Gi) FOR ROOT POOL DEVICE>
+     docker_volume_size: <REPLACE WITH THE SIZE (Gi) FOR DOCKER POOL DEVICE>
```

Comment:
The `docker_volume_size` from this dictionary doesn't seem to be honored ... I picked some "random" numbers, such as 56, 74, etc. yet all the docker volumes were created at 40Gi